src/less/tooltip/ds6/tooltip.less

```diff
@@ -0,0 +1,404 @@
+@import "../../less/ds6/variables.less";
+@import "../../less/ds6/mixins.less";
+
+@import "tooltip-base.less";
+
+/* Default style */
+.tooltip,
+.infotip,
+.tourtip {
+    display: inline-block;
+    position: relative;
+
+    &__window {
+        position: absolute;
+        z-index: 1;
+
+        &::after {
+            border-style: solid;
+            border-width: 8px;
+            content: "";
+            position: absolute;
+        }
+    }
+
+    &__body {
+        margin: 16px 16px 16px 16px;
+    }
+}
+
+.tooltip {
+    &:hover {
+        .tooltip__top-arrow,
+        .tooltip__top-leftarrow,
+        .tooltip__top-rightarrow,
+        .tooltip__bottom-arrow,
+        .tooltip__bottom-leftarrow,
+        .tooltip__bottom-rightarrow,
+        .tooltip__left-arrow,
+        .tooltip__left-toparrow,
+        .tooltip__left-bottomarrow,
+        .tooltip__right-arrow,
+        .tooltip__right-toparrow,
+        .tooltip__right-bottomarrow {
+            visibility: visible;
+        }
+    }
+
+    &__bottom-arrow,
+    &__bottom-leftarrow,
+    &__bottom-rightarrow,
+    &__top-arrow,
+    &__top-leftarrow,
+    &__top-rightarrow,
+    &__left-arrow,
+    &__left-toparrow,
+    &__left-bottomarrow,
+    &__right-arrow,
+    &__right-toparrow,
+    &__right-bottomarrow {
+        width: 344px;
+
+        &::after {
+            border-color: @ds6-color-p002-blue;
+            box-shadow: 0 0 0 @ds6-color-p002-blue;
+            box-sizing: border-box;
+        }
+    }
+
+    &__bottom-arrow,
+    &__bottom-leftarrow,
+    &__bottom-rightarrow {
+        bottom: 100%;
+        margin-bottom: 16px;
+
+        &::after {
+            top: 100%;
+        }
+    }
+
+    &__top-arrow,
+    &__top-leftarrow,
+    &__top-rightarrow {
+        margin-top: 16px;
+
+        &::after {
+            top: 2%;
+        }
+    }
+
+    &__left-arrow,
+    &__left-toparrow,
+    &__left-bottomarrow {
+        left: 110%;
+        margin-left: 16px;
+
+        &::after {
+            margin-left: -10px;
+        }
+    }
+
+    &__right-arrow,
+    &__right-toparrow,
+    &__right-bottomarrow {
+        margin-right: 16px;
+        right: 100%;
+
+        &::after {
+            left: 95%;
+            margin-left: 5px;
+        }
+    }
+
+    &__bottom-arrow,
+    &__top-arrow {
+        margin-left: -161px;
+
+        &::after {
+            left: 45%;
+            margin-left: 5px;
+        }
+    }
+
+    &__bottom-leftarrow,
+    &__top-leftarrow {
+        margin-left: -24px;
+
+        &::after {
+            left: 7%;
+        }
+    }
+
+    &__bottom-rightarrow,
+    &__top-rightarrow {
+        margin-left: -300px;
+
+        &::after {
+            left: 87%;
```

Comment:
According to the specs all these have to be fixed positions, not relative.  The ones on the right should have 8px of space between the arrow and the right edge, etc.